.pre-commit-config.yaml

```diff
@@ -39,4 +39,4 @@ repos:
       additional_dependencies:
         - flake8-docstrings
         - flake8-import-order
-      exclude: simSPI/__init__.py,tests/__init__.py
+      exclude: simSPI/__init__.py,tests/__init__.py,simSPI/linear_simulator/__init__.py
```

Comment:
this is the precommit, for when you are comiting locally and you get the automatic checks or changes (e.g. with black).

This is different from how the linting and testing run on github workflows.

Perhaps add something in here: https://github.com/compSPI/simSPI/blob/master/.github/workflows/lint.yml